checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -0,0 +1,36 @@
+# Enable StrictMode 3.0
+Set-StrictMode -Version 3.0
+
+class Config {
+    [bool]$rcc
+
+    static [string] GetConfigContent($configFilePath) {
+        try {
+            $configFileContent = Get-Content -Raw -Path $configFilePath -ErrorAction Stop
```

Comment:
What would happen if we simply wrote
`
return Get-Content -Raw -Path $configFilePath -ErrorAction Stop
`
Would this not throw the error?